cordova-lib/src/hooks/HooksRunner.js

```diff
@@ -178,7 +178,9 @@ function runScriptViaModuleLoader(script, context) {
     }
     var scriptFn = require(script.fullPath);
     context.scriptLocation = script.fullPath;
-    context.opts.plugin = script.plugin;
+    if(script.plugin) {
+        context.opts.plugin = script.plugin;
```

Comment:
Am I correct in understanding that this is already being passed in? You just add a if statement to confirm it exists before assigning?

@feichngr